webapp/src/containers/Rhs/rhsData.tsx

```diff
@@ -75,7 +75,7 @@ const RhsData = ({
     }), []);
 
     const hasMoreSubscriptions = useMemo<boolean>(() => (
-        subscriptionList.length !== 0 && (subscriptionList.length - (paginationQueryParams.page * Constants.DefaultPageSize) === Constants.DefaultPageSize)
+        (subscriptionList.length - (paginationQueryParams.page * Constants.DefaultPageSize) >= Constants.DefaultPageSize)
```

Comment:
I think it's still wrong. Correct would be - 
```suggestion
        subscriptionList.length === Constants.DefaultPageSize
```

Reply:
on thinking more I guess this logic was already correct `(subscriptionList.length - (paginationQueryParams.page * Constants.DefaultPageSize) === Constants.DefaultPageSize)` and `subscriptionList` is the total length of subscriptions, not just the list of current subscriptions fetched

Reply:
I think you should change the variable name to `totalSubscriptions`.